third_party/libevent/METADATA

```diff
@@ -6,13 +6,14 @@ description:
 third_party {
   url {
     type: GIT
-    value: "https://github.com/libevent/libevent.git"
+    value: "https://chromium.googlesource.com/chromium/src"
+    # Chromium probably uses https://github.com/libevent/libevent but see README.chromium.
   }
```

Comment:
Please still add a line that just copies the value from README.Chromium as well, so it reads like this

```
# Version: 1.4.15
version: "70.0.3538.124" # See also README.chromium.
```

I know it's duplicating information, and doesn't really track the source of truth. But there's reasonable value making things obvious at a glance for readers - and the readers are well outside of our own team. As a stopgap, it's a better status quo than what we have, until we come up with a more rigorous way of doing all of this.

Reply:
I disagree but whatever.
Let's run all the checks one more time! 🥳